crengine/src/lvdocview.cpp

```diff
@@ -4645,6 +4645,13 @@ void LVDocView::createEmptyDocument() {
     m_doc->setMinSpaceCondensingPercent(m_props->getIntDef(PROP_FORMAT_MIN_SPACE_CONDENSING_PERCENT, DEF_MIN_SPACE_CONDENSING_PERCENT));
     m_doc->setUnusedSpaceThresholdPercent(m_props->getIntDef(PROP_FORMAT_UNUSED_SPACE_THRESHOLD_PERCENT, DEF_UNUSED_SPACE_THRESHOLD_PERCENT));
     m_doc->setMaxAddedLetterSpacingPercent(m_props->getIntDef(PROP_FORMAT_MAX_ADDED_LETTER_SPACING_PERCENT, DEF_MAX_ADDED_LETTER_SPACING_PERCENT));
+    m_doc->setHangingPunctiationEnabled(m_props->getBoolDef(PROP_FLOATING_PUNCTUATION, true));
```

Comment:
Dunno if it was a typo or a decision, or if it even matters (is it always reset by frontend ? it would be the case with KOReader), but you set floating punctuation `true` (before, we had `bool gHangingPunctuationEnabled = false`), so it's enabled by default?

Reply:
I don't think that is important. Change it if you think it's necessary . Later I sync the sources.